### ISSUE TITLE
[BUG] - No cast button

### ISSUE THREAD

**Issue by ecker00** [CappielloAntonio/tempo#300]
## Describe the bug
I've downloaded the APK from github with Obtainium to get the version with casting, as I understood the F-Droid version does not include the proprietary code.
 
But I still don't see a cast button on the top of the UI. I see very little info about this difference in the repo other than old issues. Has casting been removed for both versions? 

Hope to use this with Sonos speakers. 

## Environment
 - Android device: Pixel Pro 8
 - Android OS version: Graphine OS
 - App version: 
 
Version 3.8.1
com.cappielloantonio.tempo
versionCode 25

**Comment by Serpentian** [comment]
You probably need to have remembered and active devices for cast button to appear

**Comment by CappielloAntonio** [comment]
No, the functionality is still present on the version of the app downloaded from Github. I don't know if GrapheneOS can affect this, but the functionality is still there.

**Comment by spencerwi** [comment]
I'm running stock Samsung android, have downloaded v3.9.0 via Obtanium from GitHub releases, have casted to my Chromecast devices from other apps before, and still I don't see a "cast" button.

**Comment by spencerwi** [comment]
Ah, never mind me: the Cast button appears on the "home" screen of the app, not on the track playback screen.
